docs/_docs/developers-guide/clients/jdbc-driver.adoc

```diff
@@ -72,6 +72,11 @@ jdbc:ignite:thin://host[:port][,host[:port][/schema][[?parameter1=value1][;param
 ** `partitionAwarenessMetadataCacheSize` - Size of cache to store partition awareness metadata of queries, in number of entries. Default value: `1024`.
 ** `queryTimeout` - Number of seconds the driver will wait for a `Statement` object to execute. 0 means there is no limit. Default value: `0`.
 ** `connectionTimeout` - Number of milliseconds JDBC client will wait for server to respond. 0 means there is no limit. Default value: `0`.
+** `backgroundReconnectInterval` - Background reconnect interval, in milliseconds. The value `0` can be used to disable background reconnection. Default value is `30000` (30 seconds).
```

Comment:
I don\`t like word "background" here, it`s smth from implementation and not UI related, just confused as for me. I read this description and still have a questions - what exactly changes for me from user point of view if i set it for 20 for example, or to smth huge ? Can you expand the description ?